api/base/serializers.py

```diff
@@ -609,25 +609,42 @@ class JSONAPIListSerializer(ser.ListSerializer):
 
     def to_representation(self, data):
         # Don't envelope when serializing collection
-        return [
+        errors = {}
+        bulk_skip_uneditable = self.context['request'].query_params.get('skip_uneditable', False)
+
+        if isinstance(data, collections.Mapping):
+            errors = data.get('errors', None)
+            data = data.get('data', None)
+
+        ret = [
             self.child.to_representation(item, envelope=None) for item in data
         ]
 
+        if errors and bulk_skip_uneditable:
+            ret.append({'errors': errors})
+
+        return ret
+
     # Overrides ListSerializer which doesn't support multiple update by default
     def update(self, instance, validated_data):
-        if len(instance) != len(validated_data):
-            raise exceptions.ValidationError({'non_field_errors': 'Could not find all objects to update.'})
+        bulk_skip_uneditable = self.context['request'].query_params.get('skip_uneditable', 'False')
+        if bulk_skip_uneditable == 'False':
```

Comment:
It would be better to rewrite this with 
```python
from api.base.utils import is_truthy
[...]

if not is_truthy(bulk_skip_uneditable):
```
 here, as things like `?filter_uneditable=9001` would cause this `if` to evaluate to `False`. You want to limit the ways this `if` can be `True`